Formula/n/ncurses.rb

```diff
@@ -52,6 +52,8 @@ def install
     ]
     args << "--with-terminfo-dirs=#{share}/terminfo:/etc/terminfo:/lib/terminfo:/usr/share/terminfo" if OS.linux?
 
+    ENV.append_to_cflags "-std=gnu17" if OS.linux? && DevelopmentTools.gcc_version("gcc") >= 15
```

Comment:
```suggestion
    odie "`-std=gnu17` workaround should be removed!" if build.stable? && version > "6.5"
    ENV.append_to_cflags "-std=gnu17" if OS.linux? && DevelopmentTools.gcc_version("gcc") >= 15
```
Let's add this so we don't carry this workaround forever.